client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/index.tsx

```diff
@@ -623,6 +623,7 @@ const PatternAssembler = ( {
 						onSelect={ onSelect }
 						recordTracksEvent={ recordTracksEvent }
 						onTogglePatternPanelList={ setIsPatternPanelListOpen }
+						sections={ sections }
```

Comment:
[nit] WDYT if the prop is named `selectedPatterns`? `sections` seems too nondescript.

Reply:
Sounds good 👍 Done ✅

There are other components that can use a better prop name because we use `sections={ sections }`, `patterns={ sections }`, and `sections={ patterns }`. Let's update those later.